### PR TITLE
feat: add whois command

### PR DESCRIPTION
**Pull request by FlyingHyper**
Add whois command to the bot

### CODE REVIEW

File: src/commands/whois.ts [hunk 1]
```diff
@@ -0,0 +1,60 @@
+import moment from 'moment';
+import { CommandDefinition } from '../lib/command';
+import { CommandCategory } from '../constants';
+import { makeEmbed } from '../lib/embed';
+
+const beautifiedStatus = {
```
Comment (Benjozork) [inline src/commands/whois.ts:6]: **suggestion:** Use an enum.

File: src/commands/whois.ts [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+    online: "Online",
+    idle: "Idle",
+    dnd: "Do Not Disturb",
+    offline: "Offline"
+}
+
+export const whois: CommandDefinition = {
+    name: 'whois',
+    description: 'Explain the absence of release dates or ETAs',
```
Comment (Benjozork) [inline src/commands/whois.ts:15]: **issue:** This is the wrong description :p

File: src/commands/whois.ts [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+    category: CommandCategory.PUBLIC,
+    executor: (msg) => {
+        console.log(msg.member.roles.cache.keyArray());
```
Comment (Benjozork) [inline src/commands/whois.ts:18]: **issue:** Debug output present.

**Review by Benjozork — CHANGES_REQUESTED** [review]

File: src/commands/whois.ts (hunk 2 above)
Comment (FlyingHyper) [reply src/commands/whois.ts:15]: Could you provide me a description for it?

File: src/commands/whois.ts (hunk 1 above)
Comment (FlyingHyper) [reply src/commands/whois.ts]: Changed in commit e053545

File: src/commands/whois.ts (hunk 2 above)
Comment (FlyingHyper) [reply src/commands/whois.ts]: Changed in commit d81b1f5

File: src/commands/whois.ts [hunk 4]
```diff
@@ -3,11 +3,11 @@ import { CommandDefinition } from '../lib/command';
...
-const beautifiedStatus = {
-    online: "Online",
-    idle: "Idle",
-    dnd: "Do Not Disturb",
-    offline: "Offline"
+enum beautifiedStatus {
+    online = "Online",
+    idle = "Idle",
+    dnd = "Do Not Disturb",
+    offline ="Offline"
```
Comment (Benjozork) [inline src/commands/whois.ts:10]: **style:** Enum elements should be in uppercase:

`ONLINE`, `IDLE`, etc.

**Review by Benjozork — CHANGES_REQUESTED** [review]

**Pull request merged**